microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -82,15 +82,19 @@ If Microsoft Edge detects typing in the Address bar that might contain sensitive
 <!-- ------------------------------ -->
 #### Enabling collecting Address bar diagnostic data
 
-You can configure Microsoft Edge to collect diagnostic data about the Address bar.  Collected data includes the number of offered queries for all search providers.<!-- todo: is the number of offered queries the only data that's collected through diagnostic data?  if not, shouldn't list that one only.  specify for Enterprise Search Provider & specify for all providers -->  To enable collecting diagnostic data, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Privacy, search, and services**, and then in the **Personalization & advertising** section, turn on the setting **Allow Microsoft to save your browsing activity including history, usage, favorites, web content, and other browsing data to personalize Microsoft Edge and Microsoft services like ads, search, shopping and news.**
+You can configure Microsoft Edge to collect diagnostic data about the Address bar.  Collected data includes the number of offered queries for all search providers.<!-- todo: is the number of offered queries the only data that's collected through diagnostic data?  if not, shouldn't list that one only.  specify for Enterprise Search Provider & specify for all providers -->  To enable collecting diagnostic data, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Privacy, search, and services**,<!-- todo: update as separate groups of settings --> and then in the **Personalization & advertising** section, turn on the setting **Allow Microsoft to save your browsing activity including history, usage, favorites, web content, and other browsing data to personalize Microsoft Edge and Microsoft services like ads, search, shopping and news.**
```

Comment:
did the toggle **Allow Microsoft to save your browsing activity including history, usage, favorites, web content, and other browsing data to personalize Microsoft Edge and Microsoft services like ads, search, shopping and news** end up in "Privacy and security" or "Search and services"?
@PatrizioBytes